tests/e2e/e2e_test.go

```diff
@@ -3034,5 +3034,161 @@ func TestE2E(t *testing.T) {
 
 			},
 		},
+
+		testExecution{
+			Name:       "TestKubeletConfigRemediation",
+			IsParallel: false,
+			TestFn: func(t *testing.T, f *framework.Framework, ctx *framework.Context, mcTctx *mcTestCtx, namespace string) error {
+
+				const baselineImage = "quay.io/wenshen/co_content@sha256:2c16e3d4732bdb2e87ef2833d2f1368295d96f134df805b34710cfeb62d1d028"
+				const requiredRule = "kubelet-eviction-thresholds-set-hard-imagefs-available"
+				const pbName = "upstream-ocp4"
```

Comment:
This test fails as the name `upstream-ocp4` was already used by the previous test `TestVariableTemplate`. I highly suggest making this ProfileBundle have a unique name (maybe something more relevant to the test itself) as opposed to using a hardcoded and general name, as it's making this test fail. We have a function called `getObjNameFromTest` which you could use for such purposes.

Reply:
Thanks for the advice, I will use that instead!